components/provisioner/cmd/init.go

```diff
@@ -35,7 +36,11 @@ func newProvisioningService(config config, persistenceService persistence.Servic
 	uuidGenerator := uuid.NewUUIDGenerator()
 	installationService := installation.NewInstallationService(config.Installation.Timeout, installationSDK.NewKymaInstaller, config.Installation.ErrorsCountFailureThreshold)
 
-	inputConverter := converters.NewInputConverter(uuidGenerator, releaseRepo)
+	var gardenerAccountPool hyperscaler.AccountPool = nil // TODO: get SecretInterface connected to Gardener cluster
```

Comment:
Could you please implement that part, so we can verify it is working properly? We expect to have Gardener kubeconfig in the ENV var.